server/src/database/board_overview.go

```diff
@@ -0,0 +1,28 @@
+package database
+
+import "github.com/google/uuid"
+
+func (d *Database) GetBoardOverview(id uuid.UUID) (Board, []BoardSession, []Column, error) {
+	var board Board
+	var sessions []BoardSession
+	var columns []Column
+	var err error
+
+	board, err = d.GetBoard(id)
+	if err != nil {
+		return Board{}, nil, nil, err
+	}
+
+	sessions, err = d.GetBoardSessions(id)
+	if err != nil {
+		return Board{}, nil, nil, err
+	}
+
+	columns, err = d.GetColumns(id)
+	if err != nil {
+		return Board{}, nil, nil, err
+	}
```

Comment:
Why do you return `Board{}, nil, nil, err` in the error cases here? Shouldn't that be `nil, nil, nil, err`?